torch/distributed/_tensor/examples/comm_mode_features_example.py

```diff
@@ -272,8 +272,8 @@ def example_transformer_module_tracing(self) -> None:
             output = model(inp)
 
         # print the module level collective tracing information
-        print(comm_mode.generate_comm_debug_tracing_table(noise_level=1))
-        comm_mode.log_comm_debug_tracing_table_to_file(noise_level=1)
+        print(comm_mode.generate_comm_debug_tracing_table(noise_level=0))
+        comm_mode.log_comm_debug_tracing_table_to_file(noise_level=0)
```

Comment:
This is minor, but do you want to use the default file name, or use a specific file_name as you did for "transformer_operation_log.txt"?

Reply:
I wanted to show the user that they can either put a specific name, or it will go to the default file if they don't provide one